packages/p2p-v2/src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,150 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import clsx from 'clsx';
+import { useCombobox } from 'downshift';
+import ArrowDown from '../../public/ic-chevron-down.svg';
+import reactNodeToString from '../../utils/react-node-to-string';
+import { Text } from '@deriv-com/ui/dist/components/Text';
+import TextField, { TextFieldProps } from '../TextField/TextField';
+import './Dropdown.scss';
+
+type TGenericSizes = '2xl' | '2xs' | '3xl' | '3xs' | '4xl' | '5xl' | '6xl' | 'lg' | 'md' | 'sm' | 'xl' | 'xs';
+
+type TProps = {
+    disabled?: boolean;
+    errorMessage?: TextFieldProps['errorMessage'];
+    icon?: React.ReactNode;
+    isRequired?: boolean;
+    label?: TextFieldProps['label'];
+    list: {
+        text?: React.ReactNode;
+        value?: string;
+    }[];
+    listHeight?: Extract<TGenericSizes, 'lg' | 'md' | 'sm'>;
+    name: TextFieldProps['name'];
+    onChange?: (inputValue: string) => void;
+    onSelect: (value: string) => void;
+    value?: TextFieldProps['value'];
+    variant?: 'comboBox' | 'prompt';
+};
+
+const Dropdown: React.FC<TProps> = ({
```

Comment:
Is this from Jim's PR? Or is it a different variation of it?
https://github.com/binary-com/deriv-app/pull/13120/files#diff-1c8ed0177c4044fedbac02e0b66cce4ffbf378f074d93e07d4099a32a918b4e9

Reply:
copied it from wallets.. btw dont review the pr yet.. there are conflicts.. and fixes to be done..  and i've written useravatar which is the same as avatar in ur pr.. and also repetition for getshortnickname util.. so few fixes to be done..